rpc/client.go

```diff
@@ -16,12 +22,21 @@ import (
 )
 
 // UploadReportOptions contains all the required options for uploading a report
-// to both Cedar and the Data Pipes service.
+// to both Cedar and SPS.
+
 type UploadReportOptions struct {
 	Report          *poplar.Report
-	ClientConn      *grpc.ClientConn
+	CedarClientConn *grpc.ClientConn
+	HTTPClient      *http.Client
+	SPSURL          string
 	SerializeUpload bool
 	DryRun          bool
+
+	// Whether to send the report to Cedar.
+	SendToCedar bool
+	// What fraction of requests to send to SPS.
```

Comment:
I find this description a bit confusing because this really applies to the individual report, so it is a probability of whether you are going to send the report to SPS or not. I think some like below is more clear:

```
SendRatioSPS specifies the probability of sending the report to SPS. 
If set to is 1, then the report is always sent to SPS.
```